README.md

```diff
@@ -87,7 +87,7 @@ const messages = await conversation.messages()
 await conversation.send('gm')
 // Listen for new messages in the conversation
 for await (const message of await conversation.streamMessages()) {
-  console.log(`[${message.senderAddress}]: ${message.text}`)
+  console.log(`[${message.senderAddress}]: ${message.content}`)
```

Comment:
Thank you for this @saulmc! There is one more instance of `message.text` on line 217 - just making sure - shall we update that one too? =)

Reply:
Good eye

Reply:
Tech writer habit 😅

Reply:
oooh nice one @jhaaaa!